models/topics.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+  return db.query("SELECT * FROM topics").then((result) => {
+    const topics = result.rows;
+    if (topics.length === 0) {
+      const error = new Error("Topics not found");
+      error.status = 404;
+      throw error;
+    }
```

Comment:
We actually haven't written a test for if no topics are present in the table - and I think it's not a scenario you need to check for right now. The topics are determined by us and there will always be topics in this server. I think you could remove any error handling for "topics not found" from the server